layers/meta-balena-jetson/recipes-support/hostapp-update-hooks/files/99-resin-bootfiles-forecr-dsb-ornx-orin-nano-4gb

```diff
@@ -0,0 +1,198 @@
+#!/bin/sh
+set -o errexit
+
+# Script which writes the appropriate
+# device tree with embedded cmdline
+# and updates the kernel, as well as
+# the rest of the bootloader binaries
+. /usr/libexec/os-helpers-fs
+. /usr/sbin/balena-config-vars
+source /usr/bin/uefi_common.func
+
+efivars_dir="/sys/firmware/efi/efivars/"
+# See https://github.com/OE4T/meta-tegra/blob/master/recipes-bsp/tools/setup-nv-boot-control/setup-nv-boot-control.sh.in
+platform_spec_efivar="${efivars_dir}TegraPlatformSpec-781e084c-a330-417c-b678-38e696380cb9"
+platform_compat_spec_efivar="${efivars_dir}TegraPlatformCompatSpec-781e084c-a330-417c-b678-38e696380cb9"
+os_indications_efivar="${efivars_dir}OsIndications-8be4df61-93ca-11d2-aa0d-00e098032b8c"
+tmp_file="/tmp/platformspecfile.bin"
+
+device_type="forecr-dsb-ornx-orin-nano-4gb"
```

Comment:
The device-type here should be "jetson-orin-nano-devkit" . If you flash Ubuntu + the Forecr configuration, what does /etc/nv_boot_control.conf contain for TNSPEC and COMPATIBLE_SPEC ?

Reply:
L4T installed on Nano 8GB shows this:

TNSPEC 3767-300-0003-S.1-1-1-p3768-0000-p3767-0000-a0-nvme-                     
COMPATIBLE_SPEC 3767--0003--1--jetson-orin-nano-devkit-

I'm changing to device_type to devkit then.